### PR TITLE
Update official services

### PR DESCRIPTION
**Pull request by francesco-ballarin**
See comments below.

### CODE REVIEW

File: official-services.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 The following services are officially operated by the FEniCS Project.
 
-- [Development mailing list](https://groups.google.com/forum/#!forum/fenics-dev)
```
Comment (francesco-ballarin) [inline official-services.md:5]: Has moved to slack.

File: official-services.md [hunk 2]
```diff
@@ -11,10 +10,7 @@ The following services are officially operated by the FEniCS Project.
 - [PyPi](https://pypi.org/user/fenicsproject/)
 - [Slack](https://fenicsproject.slack.com)
 - [Steering Council mailbox](mailto:fenics-steering-council@googlegroups.com)
-- [Support mailing list](https://groups.google.com/forum/#!forum/fenics-support)
```
Comment (francesco-ballarin) [inline official-services.md:14]: Has moved to slack.

File: official-services.md [hunk 3]
```diff
@@ -11,10 +10,7 @@ The following services are officially operated by the FEniCS Project.
...
-- [Twitter feed curated from #fenicsnews tag](https://twitter.com/search?q=%23fenicsnews&src=typd)
```
Comment (francesco-ballarin) [inline official-services.md:15]: Unsure if we want to keep this in the list.

File: official-services.md [hunk 4]
```diff
@@ -11,10 +10,7 @@ The following services are officially operated by the FEniCS Project.
...
 - [Ubuntu PPA](https://launchpad.net/~fenics-packages/+archive/ubuntu/fenics)
-- [UFL readthedocs.org](https://fenics.readthedocs.io/projects/ufl/en/latest/)
```
Comment (francesco-ballarin) [inline official-services.md:17]: The new versions seem to be pushed on `docs.fenicsproject.org`, while `fenics.readthedocs.io` is still at `2019.x.y`.

File: official-services.md [hunk 5]
```diff
@@ -11,10 +10,7 @@ The following services are officially operated by the FEniCS Project.
 - [PyPi](https://pypi.org/user/fenicsproject/)
 - [Slack](https://fenicsproject.slack.com)
 - [Steering Council mailbox](mailto:fenics-steering-council@googlegroups.com)
```
Comment (francesco-ballarin) [inline official-services.md:12]: Is this still active?

File: official-services.md [hunk 6]
```diff
@@ -2,7 +2,6 @@
 
 The following services are officially operated by the FEniCS Project.
 
-- [Development mailing list](https://groups.google.com/forum/#!forum/fenics-dev)
 - [Discourse support forum](https://fenicsproject.discourse.group)
 - [Docker end-user images](https://hub.docker.com/u/dolfinx)
```
Comment (francesco-ballarin) [inline official-services.md:6]: Unsure if we also want to add links to github packages.

File: official-services.md (hunk 3 above)
Comment (francesco-ballarin) [reply official-services.md:15]: (this = tweeter)

Comment (mscroggs) [reply official-services.md:15]: Agree it should be removed as API changes broke it and we have no plans to fix it

File: official-services.md (hunk 5 above)
Comment (mscroggs) [reply official-services.md:12]: This is stil active and we plan to keep it active

**Review by mscroggs — APPROVED** [review]

File: official-services.md (hunk 1 above)
Comment (francesco-ballarin) [reply official-services.md:5]: However, I just realized that that email address is the one used in every `pyproject.toml` file. If it has been discontinued, we need to use another email address there (the steering council one?).

**Review by garth-wells — APPROVED** [review]

**Pull request merged**